docs/en/operations/settings/settings.md

```diff
@@ -4768,3 +4768,15 @@ a	Tuple(
     l Nullable(String)
 )
 ```
+
+## date_time_overflow_mode {#date_time_overflow_mode}
+
+Defines behavior on overflow when converting [Date](../../sql-reference/data-types/date.md), [Date32](../../sql-reference/data-types/date32.md), [DateTime](../../sql-reference/data-types/datetime.md), [DateTime64](../../sql-reference/data-types/datetime64.md).
+
+Possible values:
+
+- `ignore` — Silently ignore overflow. The result may or may not be saturated to fit the resulting data type values range. The behavior is different for different data types.
+- `throw` — Throw an exception in case of conversion overflow.
+- `saturate` — Silently saturate the result to fit the result type values range. 
```

Comment:
Needs a better explanation what is meant by "saturate":

`Saturate the result value. If the value is smaller than the smallest value that can be represented by the target type, the result is chosen as the smallest representable value. If the value is bigger than the largest value that can be represented by the target type, the result is chosen as the largest representable value.`